p2p/Cargo.toml

```diff
@@ -28,6 +28,7 @@ helper_functions = { workspace = true }
 igd-next = { workspace = true }
 itertools = { workspace = true }
 log = { workspace = true }
+tracing ={ workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order